uelc/templates/pagetree/page.html

```diff
@@ -136,12 +136,17 @@
 
 {% if request.user.is_superuser %}
 {% else %}
+
 {% with previous=section.get_previous next_section=section.get_next %}
+
     <ul class="pager">
     {% if previous %}
+    {% get_previous_group_user_section request section as prev_sec %}
```

Comment:
Nit: "previous" or "prev_depth" could also be passed into the tag to prevent the double call to "get_previous"